packages/migration-graph-ember/src/entities/ember-addon-project-graph.ts

```diff
@@ -0,0 +1,10 @@
+import debug from 'debug';
+import { EmberAppProjectGraph } from './ember-app-project-graph';
+const DEBUG_CALLBACK = debug('rehearsal:ember-addon-project-graph');
```

Comment:
`const DEBUG_CALLBACK = debug('rehearsal:migration-graph-ember:ember-addon-project-graph');`

Reply:
^^^ if we specify scope in debug we can do things like `DEBUG=rehearsal:migration-graph-ember:*` to only debug this package